src/box/memtx_tx.c

```diff
@@ -687,7 +687,7 @@ struct tx_manager
 	 * It's the only case when we use bare mempool in memtx_tx because
 	 * we cannot account story allocation to any particular txn.
 	 */
-	struct mempool memtx_tx_story_pool[BOX_INDEX_MAX];
+	struct mempool memtx_tx_story_pool[BOX_INDEX_MAX + 1];
```

Comment:
I think this is a bug in our implementation that we index this array through `index_count`s, wasting the memory pool with index 0, and we should fix the indexing rather than the array size.